Riot/Modules/Common/Recents/RecentsViewController.m

```diff
@@ -2173,4 +2173,21 @@ - (void)roomsDirectoryCoordinatorBridgePresenterDelegateDidTapCreateNewRoom:(Roo
     self.roomsDirectoryCoordinatorBridgePresenter = nil;
 }
 
+- (void)roomsDirectoryCoordinatorBridgePresenterDelegate:(RoomsDirectoryCoordinatorBridgePresenter *)coordinatorBridgePresenter didSelectRoomWithIdOrAlias:(NSString * _Nonnull)roomIdOrAlias
+{
+    MXRoom *room = [self.mainSession vc_roomWithIdOrAlias:roomIdOrAlias];
```

Comment:
`room` will most probably nil here, for new rooms. I wonder if there is a way to peek into room, as we do for public rooms.

Reply:
The api we're using to peek into a room seems not accepting a room alias. Sorry, i was just wanted to know there was a way, you may want to remove that and save one request.

https://matrix.org/docs/spec/client_server/r0.6.1#deprecated-get-matrix-client-r0-rooms-roomid-initialsync